### PR TITLE
Many changes

### PR DESCRIPTION
**Pull request by gardusig**
Changes:
- Added new gauge metric: `task_update_time`
- Refactored `logrus` alias to be consistent through the codebase
- Refactored some errors, which were previously logged, instead of being returned
- Splitted `WorfklowExecutor` into a new class: `WorkflowMonitor`
  - It will be responsible for running workflows channels, including:
    - Creation
    - Tracking
    - Notification
- Added support for workflow execution timeout
- Added support for multiple workflow start
- Refactored integration tests:
  - Added a test for each task type
- Refactored polling interval type from `int` to `time.Duration`

### CODE REVIEW

File: pkg/conductor_client/conductor_http_client/api_client.go [hunk 1]
```diff
@@ -216,10 +216,10 @@ func (c *APIClient) mustRefreshToken() bool {
 }
 
 func (c *APIClient) refreshToken() {
-	log.Debug("Refreshing authentication token")
+	logrus.Debug("Refreshing authentication token")
 	token, response, err := c.getToken()
 	if err != nil {
-		log.Warn(
+		logrus.Warning(
```
Comment (v1r3n) [inline pkg/conductor_client/conductor_http_client/api_client.go:222]: why are we changing from log to logrus?

File: pkg/worker/task_runner.go [hunk 2]
```diff
@@ -54,101 +54,128 @@ func NewTaskRunnerWithApiClient(
...
+func (c *TaskRunner) runBatch(taskType string, executeFunction model.TaskExecuteFunction, pollInterval time.Duration, domain optional.String) (bool, error) {
+	batchSize, err := c.getAvailableWorkerAmount(taskType)
+	if err != nil {
+		return false, err
+	}
 	if batchSize < 1 {
-		sleep(1)
-		return
+		// TODO wait until there is available workers
+		time.Sleep(10 * time.Millisecond)
```
Comment (v1r3n) [inline pkg/worker/task_runner.go:144]: 10 ms is too much, let's keep it to 1 ms

**Review by v1r3n — APPROVED** [review]

**Review by v1r3n — APPROVED** [review]

File: pkg/conductor_client/conductor_http_client/api_client.go (hunk 1 above)
Comment (gardusig) [reply pkg/conductor_client/conductor_http_client/api_client.go]: Applied new standard, import `logrus` as `log` alias for entire repo

**Review by v1r3n — APPROVED** [review]

File: pkg/worker/task_runner.go (hunk 2 above)
Comment (gardusig) [reply pkg/worker/task_runner.go]: done

**Pull request merged**